src/views/profile/Detail.vue

```diff
@@ -0,0 +1,170 @@
+<script setup lang="ts">
+import { DEFAULT_NETWORK_CONFIG } from "@/helpers/config";
+import { onMounted, ref, watch } from "vue";
+import useErc725 from "@/compositions/useErc725";
+import { useRoute, RouteLocationNormalizedLoaded } from "vue-router";
+import { createIpfsLink } from "@/utils/createLinks";
+import useWeb3 from "@/compositions/useWeb3";
+
+type LSP3Profile = {
```

Comment:
We have type defined in lsp-factory, let's use it from there

Reply:
The type produced from there has a problem with version of node, I used it but its having issues

Reply:
<img width="563" alt="image" src="https://user-images.githubusercontent.com/44970013/179221531-f718d80b-c794-4c32-a233-8604bef2cedb.png">
Tests fail with ` ReferenceError: TextEncoder is not defined`

Reply:
I used a different type after speaking with Callum. I think that solves the issue